.cargo/config.toml

```diff
@@ -0,0 +1,64 @@
+[target.x86_64-pc-windows-msvc.glslang]
```

Comment:
This only applies when directly compiling this crate. It doesn't have any effect when using this crate as a dependency unfortunately. `.cargo/config.toml` is only read for the directory from which cargo was invoked, parent dirs and the home dir, not from arbitrary dependencies.